App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import {DashboardScreen} from './src/screens/dashboard/DashboardScreen';
+import {Container} from './src/components/container/Container'
 
 export const App = () => {
-  return <DashboardScreen />;
+  return <Container />;
```

Comment:
I believe the way it's supposed to be set up is you put your component onto the `DashboardScreen.js` and leave the `App.js` file to return `<DashboardScreen />`